payroll-processor.code-workspace

```diff
@@ -38,7 +38,10 @@
       "editor.defaultFormatter": "ms-dotnettools.csharp"
     },
 
-    "debug.internalConsoleOptions": "neverOpen"
+    "omnisharp.defaultLaunchSolution": "Payroll.Processor.Functions.sln",
```

Comment:
This makes the Functions solution the default project context for Omnisharp when VS Code launches